phpstan.neon

```diff
@@ -1,4 +1,8 @@
 parameters:
+    level: 6
```

Comment:
Moving this to phpstan.neon (so you can call phpstan without arguments) allows phpstan to work out of the box in PhpStorm